client/src/pages/Profile.tsx

```diff
@@ -10,10 +10,10 @@ const Profile = () => {
   const [comments, setComments] = useState([]);
 
   useEffect(() => {
-    fetch('/api/users/1')
+    fetch('/api/users/Apostolu')
```

Comment:
This is to test that it works but we want to merge the logic that will work for any user

Reply:
I thought we were waiting for Shibboleth to get set up first. Because I am not sure how to get the utorid

Reply:
You should really get the profile utorid from the url params. Lookup the `useParams()` hook.